packages/react-server-dom-webpack/src/ReactFlightWebpackNodeRegister.js

```diff
@@ -185,7 +185,6 @@ module.exports = function register() {
             );
           }: any),
           {
-            name: {value: name},
```

Comment:
If this was left in intentionally for convenience (i.e. handling in `deepProxyHandlers`), feel free to just close the PR. But including the `name` here, but not in all the other branches, felt inconsistent and confusing to me.